python-wrapper/src/neo4j_viz/neo4j.py

```diff
@@ -44,8 +45,16 @@ def from_neo4j(
         graph = result.graph()
     elif isinstance(result, neo4j.graph.Graph):
         graph = result
+    elif isinstance(result, Driver):
+        graph = result.execute_query(
+            f"MATCH (n)-[r]->(m) RETURN n,r,m LIMIT {row_limit}",
+            routing_=RoutingControl.READ,
+            result_transformer_=Result.graph,
+        )
```

Comment:
i think if we allow a driver here, we should also allow to specify a query here instead of the row_limit.
Otherwise we end up with building a Cypher query builder

Reply:
I don't think I agree with that. If they want to do their own query they can just provide the `Result`. The idea of this is to make it so that they don't even have to make a query - that's the added value

Reply:
That's a cool idea! I'll try it

Reply:
What I don't like is that it requires more parameters

Reply:
you can have a dataclass to combine url + auth. gds has `DbmsConnectionInfo` now.